docs/02.API-REFERENCE.md

```diff
@@ -1669,9 +1668,7 @@ main (void)
 
 **Summary**
 
-Parse script and construct an EcmaScript function. The lexical environment is
-set to the global lexical environment. The resource name can be used by
-debugging systems to provide line / backtrace info.
+Parse a script, module, or function and create a compiled code using a character string.
```

Comment:
We should add an example on how to parse function as this replaces `jerry_parse_function` method. Also the fact that this replaces the function parse method should be mentioned.